src/pages/index.js

```diff
@@ -2,5 +2,6 @@ import Admin from "./Admin/Admin";
 import Employee from "./Employee/Employee";
 import Login from "./Login/Login";
 import NotFound from "./NotFound/NotFound";
+import Auth from "./Auth/Auth";
 
-export { Admin, Employee, Login, NotFound };
+export { Admin, Employee, Login, NotFound, Auth };
```

Comment:
Seems like this is not an actual page. Is there any solution if I would like to get the path: `/employee`, `/admin` directly instead of `auth/employee`, `auth/admin` ?